src/Microsoft.Android.Sdk.ILLink/PreserveLists/Mono.Android.xml

```diff
@@ -21,12 +23,10 @@
 		<type fullname="Android.Runtime.RaiseThrowableEventArgs" />
 		<type fullname="Android.Runtime.RegisterAttribute" />
 		<type fullname="Android.Runtime.TypeManager" />
-		<!--
-		<type fullname="Android.Runtime.XmlResourceParserReader" />
-		<type fullname="Android.Runtime.XmlPullParserReader" />
-		<type fullname="Android.Runtime.XmlReaderResourceParser" />
-		<type fullname="Android.Runtime.XmlReaderPullParser" />
-		-->
+		<type fullname="Android.Runtime.XmlResourceParserReader" preserve="methods" />
+		<type fullname="Android.Runtime.XmlPullParserReader" preserve="methods" />
+		<type fullname="Android.Runtime.XmlReaderResourceParser" preserve="methods" />
+		<type fullname="Android.Runtime.XmlReaderPullParser" preserve="methods" />
```

Comment:
Package size increased a lot!
```
1,141,062 Package size difference 42.57% (of 2,680,724)
```
Maybe I'll try just the above two, and not these XML ones.